packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -214,18 +214,18 @@ export class GumbyController {
                 await this.initiateTransformationOnProject(message)
                 break
             case ButtonActions.CANCEL_TRANSFORMATION_FORM:
-                this.messenger.sendJobFinishedMessage(message.tabId, CodeWhispererConstants.jobCancelledChatMessage)
+                this.messenger.sendJobFinishedMessage(message.tabID, CodeWhispererConstants.jobCancelledChatMessage)
                 break
             case ButtonActions.VIEW_TRANSFORMATION_HUB:
                 await vscode.commands.executeCommand(GumbyCommands.FOCUS_TRANSFORMATION_HUB)
-                this.messenger.sendJobSubmittedMessage(message.tabId)
+                this.messenger.sendJobSubmittedMessage(message.tabID)
```

Comment:
Question: I thought we were keeping `tabId` and removing `tabID`? @JuliaABurch

Reply:
It would mean much more changes and in addition it seems tabID is more common anyhow:
`tabId` occurences: 84 in 12 files
`tabID` occurences: 891 in 43 files